db/seeds.rb

```diff
@@ -140,11 +140,11 @@ def create_roles
   end
 
   def create_admin_user_roles
-    30.times do
-      user = User.find(rand(1..100))
-      name = ["platform_admin", "charity_admin", "charity_original_admin", "registered_user"].sample
-      role = Role.find_by(name: name)
-      user.roles << role
+      User.all.each do |user|
+        name = ["platform_admin", "charity_admin", "charity_original_admin", "registered_user"].sample
```

Comment:
Let's make sure we change the name of this. It's not charity original admin in the project anymore nor are we creating a registered user role as we are only doing roles for admins. We should probably change the seed file to reflect this.